app/controllers/items_controller.rb

```diff
@@ -283,10 +283,14 @@ def show_license
   def update
     change_set = ItemChangeSet.new(@cocina)
     attributes = params.require(:item).permit(:barcode, :copyright, :use_statement, :license)
-    change_set.validate(**attributes)
-    change_set.save # may raise Dor::Services::Client::BadRequestError
-    reindex
-    redirect_to solr_document_path(params[:id]), status: :see_other
+    if change_set.validate(**attributes)
+      change_set.save # may raise Dor::Services::Client::BadRequestError
+      reindex
+      redirect_to solr_document_path(params[:id]), status: :see_other
+    else
+      logger.error "Errors: #{change_set.errors.full_messages.join}"
+      render 'error', locals: { message: message }
```

Comment:
These lines were added in this PR and are uncovered which may be hiding a bug: where is `message` defined? In this case, the 🔴 coverage among CI checks might have been telling us something worth taking a look at. @jcoyne @justinlittman

Reply:
I actually fixed this in https://github.com/sul-dlss/argo/pull/3167/files#diff-22da4de4953a8a5748f5a8f4425eb896283d008abd9525a4a33c9fd76f35f40bR294  Maybe we should pull that to it's own pr.